rules/windows/file/file_event/file_event_win_susp_right_to_left_override_extension_spoofing.yml

```
@@ -20,14 +23,32 @@ logsource:
     product: windows
 detection:
     selection_rtlo_unicode:
-        TargetFilename|contains: '\u202e'
+        TargetFilename|contains:
+            - '\u202e'  # Unicode RTLO character
+            - '[U+202E]'
+            - '%E2%80%AE'
+            - '???' # In some case Unicode cannot be represented in unicode
```

```suggestion
            - '???' # In some case Unicode cannot be represented in unicode
```
`?` is a special character in Sigma representing "any character"